keeper/integration_test.go

```diff
@@ -276,9 +282,17 @@ func (suite *KeeperTestSuite) receiverNFT(
 	token, found := toEndpoint.Chain.GetSimApp().
 		NFTKeeper.GetNFT(toEndpoint.Chain.GetContext(), classID, data.GetTokenIds()[0])
 	suite.Require().True(found, "not found class")
+
+	expToken := &nft.NFT{
+		ClassId: classID,
+		Id:      data.GetTokenIds()[0],
+		Uri:     data.GetTokenUris()[0],
+		Data:    any,
```

Comment:
is there no data.GetTokenData()? Just curious because in spec I see this: https://github.com/cosmos/ibc/pull/876/files#diff-254e967536211862be86cdfb12569fbcaa03eaf39672f9fae5ab7b226b57cc8fR327

`tokenData.push(token.GetData())`